### DISCUSSION TITLE
Filtering field values

### DISCUSSION THREAD

**Discussion by Montreal666** [grafana/grafana#61331] [Dashboards, Panels & Visualizations]
Just want to make sure I understand this correctly:

Based on reading 

this : 

https://github.com/grafana/grafana/issues/2674

and this

https://github.com/grafana/grafana/pull/12136

If I need to filter "field" values it can only be done in raw mode? and switching back to editor mode deletes the changes?
Isn't there a way to integrate some translation routine to/from the editor?
The workaround is not overly complicated but would definitely prefer this as part of the visual editor...
especially if available via the InfluxDB Explore widget/page + a number of request to implement this.

example below, can you please confirm raw is still the only way to proceed and if it is on the roadmap? 

Thanks for your help!

Need to get rid of invalid values (anything under -50 for instance) 

<img width="1615" alt="image" src="https://user-images.githubusercontent.com/56667580/211931274-dbdd8bf6-344a-42df-8f16-ab71953ff1cc.png">

Raw mode option:

<img width="1601" alt="image" src="https://user-images.githubusercontent.com/56667580/211930865-c4cacad8-25d5-4ab6-b365-dc07b16a56b0.png">

Trying to switch back to Editor:

<img width="1169" alt="image" src="https://user-images.githubusercontent.com/56667580/211930987-4b7d1f4c-37a5-47be-9dd5-f515782e6d68.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
